src/entity/Event.ts

```diff
@@ -2,6 +2,7 @@ import {Column, PrimaryGeneratedColumn, Entity, BeforeInsert, BeforeUpdate, Many
 import { validateOrReject, IsDefined } from 'class-validator';
 import {Resource} from '../entity/Resource';
 
+export type EventType = "sponsor" | "key-times" | "workshops" | "activities" | "social-events";
```

Comment:
We decided to simplify the types of events down to just 4 which means `"activities"` can be removed.